staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -127,7 +127,7 @@ type Reflector struct {
 var (
 	// We try to spread the load on apiserver by setting timeouts for
 	// watch requests - it is random in [minWatchTimeout, 2*minWatchTimeout].
-	minWatchTimeout = 5 * time.Minute
+	minWatchTimeout = 30 * time.Minute
```

Comment:
any reason using 30 minutes, i.e. not 10 minutes?
anyway, we should be careful when making such large min values, it could introduce side effort when reconnecting or concurrent socket sessions at the server.

Reply:
> any reason using 30 minutes, i.e. not 10 minutes?
> anyway, we should be careful when making such large min values, it could introduce side effort when reconnecting or concurrent socket sessions at the server.

Intentionally making big changes to see impact. What do you meant by concurrent socket sessions?

Reply:
if this is experimental, it is OK for a temp change for testing. by "socket session" i refer to the low level connections to the server for the long-get call to the api server.

Reply:
> if this is experimental, it is OK for a temp change for testing. by "socket session" i refer to the low level connections to the server for the long-get call to the api server.

I understand that part. But what is the side effect?